src/main.c

```diff
@@ -228,15 +242,15 @@ void timer0_isr() __interrupt 1 __using 1
 	    count_5000++;	//increment every 100ms
 #ifndef WITHOUT_ALARM
 	    count_20000++;	//increment every 100ms
-#endif
-#ifndef WITHOUT_CHIME
-        if (chime_trigger)
-            chime_ticks ++;	//increment every 100ms
 #endif
 	    // 2/sec: 500 ms
             if (count_5000 == 5) {
                 count_5000 = 0;
                 blinker_slow = !blinker_slow;	//blink every 500ms
+#if defined(WITH_MONTHLY_CORR) && WITH_MONTHLY_CORR != 0
+                if (!blinker_slow && corr_remaining)
```

Comment:
i don't think this is important for such an approximate adjustment. the more so since the ds1302 with an external quartz resonator is temperature dependent itself, in contrast to the DS3231, which has an internal adjustment.

earlier with my quartz resonator the clock had an accuracy of -13 seconds per month. now - less than 0.5 sec. used at a more or less constant room temperature.